project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16-M1
```

Comment:
I'm using 0.13.16-M1 because it has plugin cross building feature. It should work on any sbt version that supports auto plugin.